happi/client.py

```diff
@@ -645,6 +682,10 @@ def export(self, path=sys.stdout, sep='\t', attrs=None):
         devs = self.all_items
         logger.info('Creating file at %s ...', path)
         # Load item information
+        if attrs is None:
+            # TODO: decide how to handle None case, currently no-op
+            attrs = []
```

Comment:
Comment pre-dating PR: possibly this should an include-all-attrs instead if we care to support this function.
The fix here is better than the status quo where the function clearly errors out if you don't provide the `attrs` argument.